wafer/templates/wafer/nav.html

```diff
@@ -66,7 +66,7 @@
           </li>
           {% elif not WAFER_HIDE_LOGIN %}
           <li><a href="{% url 'auth_login' %}">
-              {% trans 'Register / Log In' %}
+              {% trans 'Sign up / Log In' %}
```

Comment:
Given the many different registration schemes possible (PyCon ZA doesn't require an account for conference registration, only for talk submission, Debconf does something different and other conferences will be different yet again), trying to sneak hints into this space isn't going to work.

Make things consistent and detailed instructions / requirements will need to be added as a page on the site.